simapp/test_helpers.go

```diff
@@ -142,13 +142,14 @@ func SetupWithGenesisValSet(t *testing.T, valSet *cmttypes.ValidatorSet, genAccs
 	)
 
 	// commit genesis changes
-	app.Commit(context.TODO(), &abci.RequestCommit{})
-	// app.BeginBlock(abci.RequestBeginBlock{Header: cmtproto.Header{
-	// 	Height:             app.LastBlockHeight() + 1,
-	// 	AppHash:            app.LastCommitID().Hash,
-	// 	ValidatorsHash:     valSet.Hash(),
-	// 	NextValidatorsHash: valSet.Hash(),
-	// }})
+	_, err = app.Commit(context.TODO(), nil)
```

Comment:
I think this needs to be removed. This will be causing consensus key nil issues

Reply:
Commit sets the `finalizeBlockState` to nil, so this makes sense.